third_party/terraform/website/docs/r/google_kms_crypto_key_iam_member.html.markdown

```diff
@@ -27,6 +27,21 @@ resource "google_kms_crypto_key_iam_member" "crypto_key" {
 }
 ```
 
+With IAM Conditions ([beta](https://terraform.io/docs/providers/google/provider_versions.html)):
+```hcl
+resource "google_kms_crypto_key_iam_member" "crypto_key" {
+  crypto_key_id = "your-crypto-key-id"
```

Comment:
Keep this consistent with the longer form?
`  crypto_key_id = "my-gcp-project/us-central1/my-key-ring/my-crypto-key"`

Reply:
Updated the docs to use more full examples that reference other resources instead of spelling out the ids